samples/OpenIdConnectSample/Startup.cs

```diff
@@ -39,6 +40,23 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
 
             app.Run(async context =>
             {
+                var authContext = new AuthenticateContext(CookieAuthenticationDefaults.AuthenticationScheme);
+                await context.Authentication.AuthenticateAsync(authContext);
+                if (context.Request.Path == new PathString("/remote-sign-out"))
+                {
+                    var session = authContext.Properties[OpenIdConnectSessionProperties.SessionState];
+                    var sid = context.Request.Query["sid"];
+                    if (string.Equals(sid, session, System.StringComparison.Ordinal))
```

Comment:
So we're all reviewing an AAD sample that relies on a feature that AAD doesn't support? #YOLO

@brockallen nah, [let's integrate ADAL directly in the OIDC middleware] (https://github.com/aspnet/Security/pull/258#discussion_r30331659) :trollface: